documentation/docs/20-core-concepts/20-load.md

```diff
@@ -420,19 +420,23 @@ export function load({ locals }) {
 
 In the browser, you can also navigate programmatically outside of a `load` function using [`goto`](modules#$app-navigation-goto) from [`$app.navigation`](modules#$app-navigation).
 
-## Promise unwrapping
+## Promises
```

Comment:
I wonder if the title should be something more descriptive so people looking for the equivalent of `+loading.svelte` or whatever can scan for it easily.

Reply:
Maybe this?

```suggestion
## Streaming with promises
```

We could add something between the two code blocks like this...

> This is useful for creating skeleton loading states:

...since 'skeleton' and 'loading states' are two things people might search for